app/models/patient.rb

```diff
@@ -641,8 +644,8 @@ def latest_transfer
   # All individuals enrolled within the given time frame
   scope :enrolled_in_time_frame, lambda { |time_frame|
     case time_frame
-    when 'Last 24 Hours'
-      where('patients.created_at >= ?', 24.hours.ago)
+    when 'Yesterday'
```

Comment:
this also needs to be updated in the `history` and `transfer` models

Reply:
good catch: https://github.com/SaraAlert/SaraAlert/pull/1085/commits/5f4d6b583878d607a0f11887bcf0739f09f58946

Reply:
might also want to make this change as well: https://github.com/SaraAlert/SaraAlert/pull/1085/files#diff-94dcc014f62f0d9b6084ad98f82f74cd40c86b029ef1691914bdbc960c2f42dcR648

Reply:
@timwongj what does this need to change to? it matches the other ones

Reply:
I think the link takes a while to load, but I was referring to the line below this one that you also changed:
```
where('patients.created_at >= ? AND patients.created_at < ?', 1.day.ago.to_date.to_datetime, Date.today.to_datetime)
```

Reply:
ah you mean the ones in `history.rb` and `transfer.rb`.  updated here: https://github.com/SaraAlert/SaraAlert/pull/1085/commits/55ee9184221cc82f340595157ed9580072b9df47